pkg/artifactory/resource_artifactory_permission_target.go

```diff
@@ -387,7 +390,9 @@ func resourcePermissionTargetDelete(d *schema.ResourceData, m interface{}) error
 
 	permissionTarget := unpackPermissionTarget(d)
 	resp, err := c.V2.Security.DeletePermissionTarget(context.Background(), *permissionTarget.Name)
-
+	if resp == nil {
```

Comment:
Check `err` first. If you check error first you probably never need to check if `resp` is `nil`.